Source/Libraries/GSF.Core/Data/DataInserter.cs

```diff
@@ -453,22 +451,23 @@ private void ResetAutoIncValues(Table table)
                 {
                     case DatabaseType.SQLServer:
                         resetAutoIncValueSQL = "DBCC CHECKIDENT('" + table.SQLEscapedName + "', RESEED)";
-                        table.Connection.ExecuteNonQuery(resetAutoIncValueSQL, Timeout);
+                        table.Connection.ExecuteNonQuery("DBCC CHECKIDENT('{0}', RESEED)", table.SQLEscapedName, Timeout);
```

Comment:
Parameter substitution does not work on Tables

Reply:
updated